src/notifications/src/index.test.ts

```diff
@@ -0,0 +1,16 @@
+const index = require('./index');
+const nodemail = require('nodemailer');
+
+jest.mock('nodemailer');
+
+const sendMailMock = jest.fn();
+nodemail.createTransport.mockReturnValue({sendMail: sendMailMock});
+

```

Comment:
I suggest that you put this all inside a beforeEach: https://jestjs.io/docs/en/setup-teardown

Reply:
Well I'm not exactly using this for other tests because I'll only be calling this once in my index.ts file which is when I want to send a notification. I do see your point if I had other tests involving it.

Reply:
It's better to get into the habit of doing this at least.